src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -0,0 +1,142 @@
+import { ProgressBar } from '../ProgressBar';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import { OverlayTrigger } from '../OverlayTrigger';
+import { Tooltip } from '../Tooltip';
+import React from 'react';
+import './UtilizationBar.css';
+
+const AvailableTooltipFunction = (max, now) => {
+  return <Tooltip id={Math.random()}>Available {max - now} %</Tooltip>;
```

Comment:
maybe we should extract this into a function? since we use it in many places we could just define it once?

further, maybe it make sense to use a timestamp vs a random, as I'm not sure how unique random will be?